sos/report/plugins/kubernetes.py

```diff
@@ -34,26 +56,32 @@ class Kubernetes(Plugin):
     ]
 
     kube_cmd = "kubectl"
-    resources = [
-        'deployments',
-        'ingresses',
-        'limitranges',
-        'pods',
-        'policies',
-        'pvc',
-        'rc',
-        'resourcequotas',
-        'routes',
-        'services'
-    ]
 
     def check_is_master(self):
         """ Check if this is the master node """
         return any(self.path_exists(f) for f in self.files)
 
     def setup(self):
-        self.add_copy_spec("/etc/kubernetes")
-        self.add_copy_spec("/run/flannel")
+        self.plugin_timeout = 1200
+        for config_file in self.config_files:
+            self.add_copy_spec(config_file)
+
+        if self.get_option('all'):
```

Comment:
I would not put this under the all, lets gather all the info by default as this is quite useful for us to have.

Reply:
Yup. This is changed.